owncloud/test/test.py

```diff
@@ -582,6 +588,30 @@ def test_share_with_link(self, file_name):
         self.assertTrue(type(share_info.get_link()) is str)
         self.assertTrue(type(share_info.get_token()) is str)
 
+    @data_provider(files)
+    def test_share_with_link_greater_8_2(self, file_name):
+        """Test sharing a file with link"""
+
+        current_version = self.client.get_version().split('.')
+        current_version_major = current_version[0]
+        current_version_minor = current_version[1]
+        if ((current_version_major < '8') or ((current_version_major == '8') and (current_version_minor < '2'))):
```

Comment:
should the string be converted to an int first ? Not sure what the comparison does, maybe byte-wise ?